app/views/code_reviews/index.html.erb

```diff
@@ -1,19 +1,17 @@
 <h1>
   Code Reviews
+  <% if can? :manage, CodeReview %>
+    <%= link_to '+ New Code Review', new_code_review_path, class: 'text-success new-link' %>
+  <% end %>
```

Comment:
You might make a partial to DRY up the headers so you don't have to change
them everywhere.

On Friday, September 18, 2015, Christopher Joy-Webb <
notifications@github.com> wrote:

> In app/views/code_reviews/index.html.erb
> <https://github.com/epicodus/epicenter/pull/189#discussion_r39886124>:
>
> > @@ -1,19 +1,17 @@
> >  <h1>
> >    Code Reviews
> > +  <% if can? :manage, CodeReview %>
> > +    <%= link_to '+ New Code Review', new_code_review_path, class: 'text-success new-link' %>
> > +  <% end %>
>
> Also, the same applies for new internships and companies. I'm trying to
> keep the design consistent, whatever it ends up being.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/epicodus/epicenter/pull/189/files#r39886124>.
>